assignments/pull_requests.go

```diff
@@ -0,0 +1,98 @@
+package assignments
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"strings"
+
+	pb "github.com/autograde/quickfeed/ag"
+	"github.com/autograde/quickfeed/database"
+	"github.com/autograde/quickfeed/scm"
+	"github.com/google/go-github/v35/github"
+	"go.uber.org/zap"
+)
+
+var ErrInvalidBody = errors.New("invalid pull request body")
+
+// getLinkedIssue returns the issue number from a pull requests body.
+// E.g. 30, from the body "Fixes #30".
+func getLinkedIssue(body string) (uint64, error) {
+	if count := strings.Count(body, "#"); count != 1 {
```

Comment:
Thinking aloud... So it is a requirement that the PR is only associated with one issue... That makes sense. However, I think the PR body could contain the `#` symbol followed by a space: `# ` indicating a markdown heading.

Reply:
I am not sure I follow you here.
The reason I wanted the body formatted like this was so that GitHub automatically linkes the issue.
That way, the issue is closed naturally when the PR is closed.

Reply:
I also want the issue linked so that we can delete the issue db-record when the PR is correctly merged.